tests/release/e2e-test-default-bundle.go

```diff
@@ -79,10 +83,35 @@ var _ = framework.ReleaseSuiteDescribe("[HACBS-738]test-release-service-default-
 			},
 		}
 
+		// using cdq since git ref is not known
+		compName = componentName
+		var componentDetected appservice.ComponentDetectionDescription
+		cdq, err := fw.AsKubeAdmin.HasController.CreateComponentDetectionQuery(compName, devNamespace, gitSourceComponentUrl, "", "", "", false)
+		Expect(err).NotTo(HaveOccurred())
+		Expect(len(cdq.Status.ComponentDetected)).To(Equal(1), "Expected length of the detected Components was not 1")
+
+		for _, compDetected := range cdq.Status.ComponentDetected {
+			compName = compDetected.ComponentStub.ComponentName
+			componentDetected = compDetected
+		}
+
 		_, err = fw.AsKubeAdmin.ReleaseController.CreateReleasePlan(sourceReleasePlanName, devNamespace, applicationNameDefault, managedNamespace, "")
 		Expect(err).NotTo(HaveOccurred())
 
-		_, err = fw.AsKubeAdmin.ReleaseController.CreateReleaseStrategy(releaseStrategyDefaultName, managedNamespace, releasePipelineNameDefault, constants.ReleasePipelineImageRef, releaseStrategyPolicyDefault, releaseStrategyServiceAccountDefault, paramsReleaseStrategyMvp)
+		sc := fw.AsKubeAdmin.ReleaseController.GenerateReleaseStrategyConfig(compName, releasedImagePushRepo)
+		scYaml, err := yaml.Marshal(sc)
+		Expect(err).ShouldNot(HaveOccurred())
+
+		scPath := "mvp-demo.yaml"
```

Comment:
Would it make sense to change the name of this file, i.e. to something like `release-default-bundle.yaml` and similarly in other release tests?
([here](https://github.com/redhat-appstudio/e2e-tests/pull/500/files#diff-0ef984b13662fe17a6a6a36aadf17220299b09e9af0e7e59b3f9ebe6f2dfd2e8R99) and [here](https://github.com/redhat-appstudio/e2e-tests/pull/500/files#diff-72c9867253b06257b82c980da3eca26e313da4af492b24946dae749ad74a972dR139))

Reply:
Sure! Ty